lib/node_modules/@stdlib/utils/parse-ndjson/lib/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+/**
+* Parse a string as Newline-Delimited JSON (NDJSON).
+*
+* @module @stdlib/utils/parse-ndjson
+*
+* @example
+* var parseNDJSON = require( '@stdlib/utils/parse-ndjson' );
+*
+* var arr = parseNDJSON( '{"beep":"boop"}\n{"example":42}\n' );
+* // returns [{'beep':'boop'}, {'example':42}]
```

Comment:
```suggestion
* // returns [ { 'beep': 'boop' }, { 'example': 42 } ]
```

In general, prefer `stdlib` style spacings, even in doctest comments.